### PR TITLE
Final changes for FMS v0.2

### PR DESCRIPTION
**Pull request by v-dobrev**
- [x] Clean up compiler warnings
- [x] Setup basic CI
- [x] Apply style
- [x] Update version variables
- [x] Update `doc/components.png`
- [x] Update copyright statements
- [x] Add CHANGELOG
- [x] Mention VisIt support in README, documentation, etc
- [x] Update the release date in CHANGELOG, etc

### CODE REVIEW

File: .github/workflows/build-and-test.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+#
+# The CEED research is supported by the Exascale Computing Project (17-SC-20-SC)
+# a collaborative effort of two U.S. Department of Energy organizations (Office
+# of Science and the National Nuclear Security Administration) responsible for
+# the planning and preparation of a capable exascale ecosystem, including
+# software, applications, hardware, advanced system engineering and early
+# testbed platforms, in support of the nation's exascale computing imperative.
+
+name: Build and test
+
```
Comment (tzanio) [inline .github/workflows/build-and-test.yaml:18]: Thanks for adding GH Actions and the style check!

File: CMakeLists.txt [hunk 2]
```diff
@@ -17,8 +17,9 @@
 # We use the C_STANDARD target property introduced in CMake 3.1
 cmake_minimum_required (VERSION 3.1)
 
-# Project name
-project(FMS VERSION 0.1 LANGUAGES C)
+# Project name and version.
+# The version is also defined in the files: fms.h and Doxyfile.
+project(FMS VERSION 0.2 LANGUAGES C)
```
Comment (tzanio) [inline CMakeLists.txt:22]: 🎉 🎉

**Review by tzanio — APPROVED** [review]
This looks great, thanks @v-dobrev !

Lets 🚢  it!

**Comment by v-dobrev** [comment]
@BradWhitlock and @Laganellac, if you want, take a look at this and let us know if you have any comments.

**Comment by v-dobrev** [comment]
@cyrush, I managed to test this FMS branch (as v0.2 release candidate) with mfem v4.3 in the latest development branch of VisIt and everything seemed to work fine. If you want, give it a try too. We plan to release this soon, probably on Friday.

File: src/fmsio.c [hunk 3]
```diff
@@ -82,8 +87,14 @@ static void FmsErrorDebug(int err_code, const char *func, const char *file,
...
 }
-#ifndef _MSC_VER
+// __PRETTY_FUNCTION__ is the same as the standard __func__ for most compilers,
+// with the exception of clang. GCC 9.3.0 with -Wpedantic gives a warning about
+// __PRETTY_FUNCTION__ not being standard, so we use __PRETTY_FUNCTION__ only
+// with clang.
+#if defined(__clang__)
 #define FMS_PRETTY_FUNCTION __PRETTY_FUNCTION__
+#elif !defined(_MSC_VER)
+#define FMS_PRETTY_FUNCTION __func__
```
Comment (cwsmith) [inline src/fmsio.c:97]: Would adding a header `fmsfname.h` to define `__func__` and friends make sense?  It looks like these definitions are also in `fms.c`.

**Review by cwsmith — APPROVED** [review]
This looks good to me. 👍 There is one small comment below.

**Comment by cyrush** [comment]
@v-dobrev  I won't get a chance to try until after friday, however if you were able to test I think we are probably in good shape.

File: src/fmsio.c (hunk 3 above)
Comment (v-dobrev) [reply src/fmsio.c:97]: Right now there just a few things that are shared by `fms.c` and `fmsio.c`, so I don't see this as something urgent.

In the future, we can factor common parts in a private header (used during building but not installed).

**Comment by tzanio** [comment]
I think we should 🚢 this

**Pull request merged**